### PR TITLE
Add support for selecting model type

### PR DESCRIPTION
**Pull request by swfz**
I want to use gpt-4.

### CODE REVIEW

File: commands/main.ts [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 export class MainCommand extends Command {
   constructor() {
     super();
+
+    const modelType = new EnumType(["gpt-3.5-turbo", "gpt-4"]);
+
     this.name("cg")
       .env("OPENAI_ACCESS_TOKEN=<value:string>", "OPENAI ACCESS TOKEN")
       .env("OPENAI_API_KEY=<value:string>", "OPENAI API KEY")
+      .type("model-type", modelType)
```
Comment (ytkg) [inline commands/main.ts:14]: Is it possible to change to this?

```suggestion
      .type("model-type", new EnumType(["gpt-3.5-turbo", "gpt-4"]))
```

Comment (swfz) [reply commands/main.ts:14]: Seems inapplicable because the test is failing.

image

Comment (ytkg) [reply commands/main.ts:14]: Maybe it's a bug?

**Review by ytkg — APPROVED** [review]
LGTM!!

**Comment by ytkg** [comment]
It might be better if it could be specified in an environment variable.

File: commands/main.ts (hunk 1 above)
Comment (swfz) [reply commands/main.ts:14]: I haven't looked into it that much

Comment (ytkg) [reply commands/main.ts:14]: https://github.com/c4spar/deno-cliffy/issues/461
https://github.com/c4spar/deno-cliffy/pull/573

It appears that a fixed version has not yet been released.

**Pull request merged**